install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -234,6 +224,40 @@ spec:
               fi
 {{- end }}
 
+              # AKS: If azure-vnet is installed on the node, and (still) configured in bridge mode,
+              # configure it as 'transparent' to be consistent with Cilium's CNI chaining config.
+              # If the azure-vnet CNI config is not removed, kubelet will execute CNI CHECK commands
+              # against it every 5 seconds and write 'bridge' to its state file, causing inconsistent
+              # behaviour when Pods are removed.
+              if stat /etc/cni/net.d/10-azure.conflist > /dev/null 2>&1; then
+
+                echo "azure-vnet configured in bridge mode. Changing to 'transparent'.."
```

Comment:
Nit: Personal preference for three dots in ellipses, in particular to remain consistent with the other messages in the file.

Reply:
Fixed.